lib/src/main/java/io/ably/lib/realtime/ChannelStateListener.java

```diff
@@ -7,6 +7,10 @@
  */
 public interface ChannelStateListener {
 
+    /**
+     * Called when channel state changes.
+     * @param stateChange information about the new state. Check {@link ChannelState ChannelState} - for all states available.
+     */
```

Comment:
This documentation is slightly out of scope of this PR. Can you update the PR / issue description for this?